### PR TITLE
[now-build-utils][now-cli] Fix `isOfficialRuntime()` edge case bug

### PR DESCRIPTION
**Pull request by TooTallNate**
Fixes a bug where `isOfficialRuntime('static', '@now/static-build')` was returning `true` when it should have been `false`.

And use the function from `@vercel/build-utils` in Vercel CLI.

### CODE REVIEW

File: packages/now-build-utils/src/index.ts [hunk 1]
```diff
@@ -85,10 +85,14 @@ export * from './errors';
...
   return (
-    typeof name === 'string' &&
-    (name.startsWith(`@vercel/${desired}`) ||
-      name.startsWith(`@now/${desired}`))
+    name === `@vercel/${desired}` ||
+    name === `@now/${desired}` ||
+    name.startsWith(`@vercel/${desired}@`) ||
+    name.startsWith(`@now/${desired}@`)
   );
 };
```
Comment (styfle) [inline packages/now-build-utils/src/index.ts:97]: Do we want to add `isStaticRuntime` or is that overkill?

```
export const isStaticRuntime = (name?: string): boolean => {
	return isOfficialRuntime('static', name);
};
```

**Review by styfle — APPROVED** [review]
Nice! It looks like this fixes a bug where `isOfficialRuntime('static', '@now/static-build')` was returning `true` when it should have been false.

Should we also add some unit tests?

**Comment by TooTallNate** [comment]
@styfle Ya that's precisely the bug.

File: packages/now-build-utils/src/index.ts (hunk 1 above)
Comment (TooTallNate) [reply packages/now-build-utils/src/index.ts:97]: Done.

**Review by styfle — APPROVED** [review]

**Pull request merged**